framework/src/utils/FormattedTable.C

```diff
@@ -609,7 +609,8 @@ FormattedTable::getTermWidth(bool use_environment) const
       ss >> w.ws_col;
     }
   }
-  else
+  // Default to AUTO if no environment variable was set
+  if (!use_environment || (w.ws_col == std::numeric_limits<unsigned short>::max()))
```

Comment:
If `use_environment` is `false`, then `w.ws_col` is guaranteed to be `std::numeric_limits<unsigned short>::max()`, right? Anyway I think we can simplify this
```suggestion
  if (w.ws_col == std::numeric_limits<unsigned short>::max())
```

Reply:
that works